test/unit/test-amp-story-player.js

```diff
@@ -370,5 +370,80 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
         'Story URL not found in the player: https://example.com/story6.html'
       );
     });
+
+    it('back button should be created', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      playerEl.setAttribute('button', 'back');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      expect(playerEl.shadowRoot.querySelector('a')).to.exist;
+    });
+
+    it('close button should be created', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      playerEl.setAttribute('button', 'close');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      expect(playerEl.shadowRoot.querySelector('a')).to.exist;
+    });
+
+    it('no button should be created', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      playerEl.setAttribute('button', 'brokenattribute');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      // Previous implementation relied on a ternary statement that defaulted to back-button
```

Comment:
Having the test should suffice, no explanation about previous implementations / bugs is necessary unless it's a highly specific / esoteric case. Adding comments should only be used as a last resort if your code can't properly communicate something.